docker/server/Dockerfile

```diff
@@ -19,7 +19,7 @@ COPY Makefile Makefile
 RUN make build_client
 
 
-FROM library/golang:1.22.1-alpine as server
+FROM library/golang:1.22.1-alpine AS server
```

Comment:
Side remark: 1.22.1 is pretty old. Is there an image for say 1.22.9?

Reply:
Updated to the latest stable.